src/events/music/musicRemove.ts

```diff
@@ -14,7 +14,7 @@ export default class extends Event {
 		}
 
 		for (const subscription of manager.websocketUserIterator()) {
-			subscription.send({ action: OutgoingWebsocketAction.MusicRemove, data: { id: song.id } });
+			subscription.send({ action: OutgoingWebsocketAction.MusicRemove, data: { queue: manager.queue.map(s => s.toJSON()) } });
```

Comment:
Hear me out: Why is musicRemove and musicSongFinish returning the full queue? Surely it isn't that hard on the frontend to `queue.filter(s => s.id != packet.data.id)`.. At this point, you're sending the same payload through a lot of events, and might as well unify it into a MusicSongUpdate generic one..

Reply:
I thought so as well but for some godforsaken reason the frontend completely loses track of what the previous queue was when receiving new data and I've tried literally everything under the sun to resolve it so I ended up with this.

Believe me, if it could be done I would not have made this change.

Reply:
Link me the code that's affected and let me take a quick peek at it 👀